### PR TITLE
Add http request from client

### PR DESCRIPTION
**Pull request by kamontia**

### CODE REVIEW

**Comment by kamontia** [comment]
@chaspy  Please review and merge. The test is to check whether this branch is deleted automatically or not. Wao.

File: main.go [hunk 1]
```diff
@@ -1,54 +1,66 @@
...
-				FullName string `json:"full_name"`
-			} `json:"repo"`
-		} `json:"head"`
-	} `json:"pull_request"`
+type GithubKeys struct {
+	AccessToken string `json:"GITHUB_ACCESS_TOKEN"`
 }
 
 func delete_branch(event PullRequestEvent) (MyResponse, error) {
 	fmt.Println("log start")
```
Comment (chaspy) [inline main.go]: please remove unncessary logs.

Comment (kamontia) [reply main.go]: Removed unncessary logs.

**Comment by chaspy** [comment]
🎉

**Pull request merged**